python/tvm/topi/rocm/tensor_intrin.py

```diff
@@ -0,0 +1,148 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, unnecessary-lambda, too-many-arguments
+"""MFMA Tensor intrinsics for GFX908."""
+import tvm
+from tvm import te
+
+
+def intrin_mfma_load_matrix(shape, matrix, thread=None, strides_src=None, strides_dst=None):
+    """Intrin function for loading thread registers for mfma tensorization"""
+    M, N, K = shape
+    assert M == 16
+    assert N == 16
+    assert K == 16
+    if matrix in ("A", "BT", "W"):
+        row, col = M, K
+    elif matrix == "B":
+        row, col = K, N
+    output_shape = (row, col)
+
+    A = te.placeholder(output_shape, name=matrix, dtype="float16")
+    BA = tvm.tir.decl_buffer(A.shape, A.dtype, scope="shared", offset_factor=1, strides=strides_src)
+
+    C = te.compute(output_shape, lambda i, j: A[i, j], name="C")
+
+    BC = tvm.tir.decl_buffer(C.shape, C.dtype, scope="local", offset_factor=1, strides=strides_dst)
+
+    def intrin_func(ins, outs):
+        ib = tvm.tir.ir_builder.create()
+
+        BA = ins[0]
+        BC = outs[0]
+
+        tx = thread
+        if tx is None:
+            tx = te.thread_axis("threadIdx.x")
+            ib.scope_attr(tx, "thread_extent", 64)
+
+        blk_td = tx % 16
+        offset = tx // 16
+        # TODO(csullivan): Using offset works, but using tx directly does not, fix this
```

Comment:
How about `tx + 0`? I've hit a similar issue and that was my workaround:
https://github.com/apache/tvm/blob/f4ec5fd4ae346dbdd8e915c048aeed94b44f6776/python/tvm/topi/cuda/nms.py#L358